src/input/src/simulations/mc.rs

```diff
@@ -20,6 +20,12 @@ impl FromTomlWithData for MonteCarlo {
 
         let mut mc = MonteCarlo::new(temperature);
 
+        if config.get("update_delta_every").is_some() {
+            let update_amplitude = try!(extract::uint(
+                    "update_delta_every", config, "Monte-Carlo propagator"));
+            mc.set_amplitude_update_frequency(update_amplitude);
```

Comment:
I would use the same name in input and code. Either `update_delta_every` or `update_frequency` (I prefer the second), but the same one.

This help users of the code to dive in and write rust code: they are already familiar with the naming of functions.